src/main/java/InputBoundary/newPlaylistInputBoundary.java

```diff
@@ -2,5 +2,9 @@
 import InputData.newPlaylistInputData;
 
 public interface newPlaylistInputBoundary {
-    public String newPlaylist(newPlaylistInputData inputData);
+    void newPlaylist(newPlaylistInputData inputData);
+
+    default String showConfirmation(){
```

Comment:
This should not be a default method - this means that things that implement this interface can use this 'default' version of the method, which would be unwanted.

Instead, you should have it be public, which will also require replacing the braces with a semicolon.